librz/debug/p/native/linux/reg/linux-loongarch64.h

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2016 dromero <dromero.perez@hotmail.com>
```

Comment:
Please set your copyright.

Reply:
also fix the year

Reply:
I have updated the code. Thank you for your suggestion